pkgs/tools/text/morsel/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "morsel";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "SamLee514";
+    repo = "morsel";
+    rev = "237fef779999718c2f7d1d93f89f7cdc885e0092";
```

Comment:
Why don't you use the tag instead of a hash?

Reply:
Because the license file didn't exist at the tag

Reply:
The license file isn't installed in the end, I don't think it matters much. 

You should at the very least add a comment explaining why it's a commit hash, for anyone looking at the code to not get confused like me.

Reply:
Switched to tag.